### PR TITLE
Rename text tooltip -> text selection menu

### PR DESCRIPTION
**Pull request by grahamlangford**
## What does this PR do?

- Renames all instances of textTooltip and similar to textSelectionMenu

## Future Work

- Same work for the commandPopover -> shortcutSnippetMenu

## Checklist

- [x] Designate a primary reviewer @fungairino

### CODE REVIEW

File: src/contentScript/textSelectionMenu/selectionMenuController.tsx [hunk 1]
```diff
@@ -40,18 +39,16 @@ import { prefersReducedMotion } from "@/utils/a11yUtils";
 import { getSelectionRange } from "@/utils/domUtils";
 
 import { snapWithin } from "@/utils/mathUtils";
+import ActionRegistry from "@/contentScript/textSelectionMenu/ActionRegistry";
 
 const MIN_SELECTION_LENGTH_CHARS = 3;
 
 export const tooltipActionRegistry = new ActionRegistry();
```
Comment (fungairino) [inline src/contentScript/textSelectionMenu/selectionMenuController.tsx:46]: Do we want to rename this to `selectionMenuActionRegistry`?

File: src/contentScript/textSelectionMenu/selectionMenuController.tsx [hunk 2]
```diff
@@ -129,40 +126,40 @@ async function showTooltip(): Promise<void> {
  * Hide the tooltip. Safe to call multiple times, even if the tooltip is already hidden.
  */
 function hideTooltip(): void {
```
Comment (fungairino) [inline src/contentScript/textSelectionMenu/selectionMenuController.tsx:128]: rename to `hideMenu`? same for `create*` and `destroy*`

File: src/contentScript/textSelectionMenu/selectionMenuController.tsx [hunk 3]
```diff
@@ -236,34 +233,30 @@ async function updatePosition(): Promise<void> {
...
-      );
-      Object.assign(selectionTooltip.style, {
+      const { x, y } = await computePosition(referenceElement, selectionMenu, {
+        placement: "bottom",
+        strategy: "fixed",
+        // `inline` prevents from appearing detached if multiple lines selected: https://floating-ui.com/docs/inline
+        middleware: [
+          ...(supportsInline ? [inline()] : []),
+          offset(10),
+          // Using flip/shift to ensure the tooltip is visible in editors like TinyMCE where the editor is in an
```
Comment (fungairino) [inline src/contentScript/textSelectionMenu/selectionMenuController.tsx:250]: update mentions of "tooltip" in comments?

**Review by fungairino — APPROVED** [review]

File: src/platform/platformProtocol.ts [hunk 4]
```diff
@@ -164,7 +164,7 @@ export interface PlatformProtocol {
    * The registry for the text selection tooltip.
```
Comment (fungairino) [inline src/platform/platformProtocol.ts:164]: tooltip -> menu

File: src/contentScript/textSelectionMenu/selectionMenuController.tsx (hunk 1 above)
Comment (grahamlangford) [reply src/contentScript/textSelectionMenu/selectionMenuController.tsx]: I think ActionRegistry is ok. But I can change it if you feel strongly.

File: src/contentScript/textSelectionMenu/selectionMenuController.tsx (hunk 2 above)
Comment (grahamlangford) [reply src/contentScript/textSelectionMenu/selectionMenuController.tsx]: Already fixed.

File: src/contentScript/textSelectionMenu/selectionMenuController.tsx (hunk 3 above)
Comment (grahamlangford) [reply src/contentScript/textSelectionMenu/selectionMenuController.tsx]: Already fixed. Let me know if you see any others.

File: src/platform/platformProtocol.ts (hunk 4 above)
Comment (fungairino) [inline src/platform/platformProtocol.ts:164]:
```suggestion
   * The registry for the text selection menu.
```

**Pull request merged**